integration/integration.go

```diff
@@ -293,24 +293,28 @@ func (i *Infrastructure) initNWO() {
 		switch label {
 		case "fsc":
 			// treat fsc as special
-			fscTopology = topology
-			continue
+			factory, ok := i.PlatformFactories["fsc"]
+			Expect(ok).To(BeTrue(), "expected to find platform [%s]", label)
+
+			fcsPlatform := factory.New(i.Ctx, topology, i.BuildServer.Client())
+			platforms = append(platforms, fcsPlatform)
+			i.FscPlatform, ok = fcsPlatform.(*fsc.Platform)
+			if !ok {
+				logger.Warnf("something went wrong here ...")
+			}
 		default:
+			logger.Infof("Register %s", label)
 			factory, ok := i.PlatformFactories[label]
 			Expect(ok).To(BeTrue(), "expected to find platform [%s]", label)
 			platforms = append(platforms, factory.New(i.Ctx, topology, i.BuildServer.Client()))
 		}
 	}
-	// Add FSC platform
-	fcsPlatform := i.PlatformFactories["fsc"].New(i.Ctx, fscTopology, i.BuildServer.Client()).(*fsc.Platform)
-	platforms = append(platforms, fcsPlatform)
```

Comment:
I think the point here was to have fscPlatform as the last one maybe

Reply:
ah good point